content/en/docs/languages/php/sdk.md

```diff
@@ -130,6 +130,7 @@ There are also a number of PHP-specific configurations:
 | `OTEL_PHP_INTERNAL_METRICS_ENABLED`  | `false`       | `true`, `false`                                                                       | `true`                       | Whether the SDK should emit metrics about its internal state (for example, batch processors) |
 | `OTEL_PHP_DISABLED_INSTRUMENTATIONS` | `[]`          | Instrumentation names, or `all`                                                       | `psr15,psr18`                | Disable one or more installed auto-instrumentations                                          |
 | `OTEL_PHP_EXCLUDED_URLS`             | ``            | Comma-delimited regular expression patterns                                           | `client/.*/info,healthcheck` | Do not load the SDK if request URL matches one of the supplied regular expressions           |
+| `OTEL_PHP_DEBUG_SCOPES_DISABLED`     | `false`       | `true`, `false`                                                                       | `true`                       | Enable/disable Scope detachment debugging (highly recommended for production environments). |
```

Comment:
```suggestion
| `OTEL_PHP_DEBUG_SCOPES_DISABLED`     | `false`       | `true`, `false`                                                                       | `true`                       | Turn on or off scope detachment debugging. Recommended for production environments. |
```

Reply:
I actually reverted the recommendation, since I think I mistook that for something else. I'll adjust either way.